FFmpegInterop/FFmpegInteropMSS.cpp

```diff
@@ -1549,7 +1645,8 @@ void FFmpegInteropMSS::OnSampleRequested(Windows::Media::Core::MediaStreamSource
 		}
 		else if (currentVideoStream && args->Request->StreamDescriptor == currentVideoStream->StreamDescriptor)
 		{
-			args->Request->Sample = currentVideoStream->GetNextSample();
+			auto sample = currentVideoStream->GetNextSample();
```

Comment:
This probably needs to be reverted, I just separated it to be able to put a breakpoint and detect null samples easier.